packages/ui/src/ui/components/formatters/FormattedLink.js

```diff
@@ -7,7 +7,7 @@ import Link from '../../components/Link/Link';
 import FormattedText from './FormattedText';
 
 function FormattedLink(props) {
-    const {state, theme, className, text, match, onClick, ...rest} = props;
+    const {state, theme = 'ghost', className, text, match, onClick, ...rest} = props;
```

Comment:
I didn't quite understand the approach. In packages/ui/src/ui/components/OperationPool/OperationPool.tsx, you transfer destructuring assignment to the attributes of the function, but not here

Reply:
In OperationPool.tsx the line prettified to several lines, but not here.